ld: symbol(s) not found for architecture x86_64

When i try to install node-vad, the error is raised. Thanks.

OS : osx 10.14(Mojave)

bindings.o
  "v8::Value::ToInt32(v8::Local<v8::Context>) const", referenced from:
      vad::vadAlloc_(Nan::FunctionCallbackInfo<v8::Value> const&) in vad_bindings.o
      vad::(anonymous namespace)::VADWorker::HandleOKCallback() in vad_bindings.o
  "v8::External::Value() const", referenced from:
      Nan::imp::FunctionCallbackWrapper(v8::FunctionCallbackInfo<v8::Value> const&) in vad_bindings.o
  "_node_module_register", referenced from:
      _register_vad() in vad_bindings.o
  "_uv_default_loop", referenced from:
      vad::vadProcessAudioBuffer_(Nan::FunctionCallbackInfo<v8::Value> const&) in vad_bindings.o
  "_uv_queue_work", referenced from:
      vad::vadProcessAudioBuffer_(Nan::FunctionCallbackInfo<v8::Value> const&) in vad_bindings.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [Release/vad.node] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/test/.nvm/versions/node/v8.9.4/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at emitTwo (events.js:126:13)
gyp ERR! stack     at ChildProcess.emit (events.js:214:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
gyp ERR! System Darwin 18.0.0
gyp ERR! command "/Users/test/.nvm/versions/node/v8.9.4/bin/node" "/Users/test/.nvm/versions/node/v8.9.4/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/test/nodetest/node_modules/node-vad
gyp ERR! node -v v8.9.4
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok
npm WARN grunt-ssh@0.12.9 requires a peer of grunt@~0.4.1 but none is installed. You must install peer dependencies yourself.

--- a/binding.gyp
+++ b/binding.gyp
@@ -3,7 +3,7 @@
         {
             'target_name': 'vad',
             'product_extension': 'node',
-            'type': 'shared_library',
+            'type': 'loadable_module',
             'defines': [],
             'include_dirs': ["<!(node -e \"require('nan')\")", "./src"],
             'sources': [

